heralding/capabilities/ssh.py

```diff
@@ -65,6 +70,13 @@ def handle_connection(self):
         else:
             self.session = self.create_session(self.address, self.dest_address)
 
+    def get_auxiliary_info(self):
+        conn = self.connections_list[-1]
```

Comment:
This is a flaw. Consider the following chain of events:

1. Client 1 connects  with MAC hmac-sha1.
2. Client 2 connects with MAC umac-64-etm blah@openssh.com.
3. Client 1 disconnects.
4. Client 2 disconnects.

In this case when reviewing the log, you will find that both sessions has a _recv_mac_ with the value _umac-64-etm@openssh.com_ which is wrong.